src/linear_solver_routines/block_col_projection.jl

```diff
@@ -0,0 +1,53 @@
+# This file is pat of RLinearAlgebra.jl
+"""
+    LinSysBlkColGent <: LinSysBlkColProjection
+
+A mutable structure that represents a standard block column projection method.
```

Comment:
This is not sufficiently descriptive

Reply:
I have added more descriptions and references.